frontend/components/Overlay/CreateProject.js

```diff
@@ -1,87 +1,31 @@
-import { useState } from 'react';
-import classes from '../../styles/Overlay.module.css';
-import Modal from './Modal';
-import Backdrop from './Backdrop';
-import Button from '../UI/Button';
-import Worldcoin from './Worldcoin';
+import { useState } from "react";
+import Modal from "./Modal";
+import Backdrop from "./Backdrop";
+import Worldcoin from "./Worldcoin";
```

Comment:
We won't use Worldcoin so this component can be deleted and add the Lens authentication